### ISSUE TITLE
1.3-ATM-Stanley

### ISSUE THREAD

**Issue by mcmillanator** [tiy-chs-ruby/assignments-june-2015#20]
https://github.com/mcmillanator/1.3-ATM/pull/5
### Tasks
- [x] Create a new Issue with these tasks!
- [x] Use the Github web interface to create a repository called `1.2-Countries`
- [x] Use the Github web interface to create a branch on `1.2-Countries` called `dev`
- [x] In Github, open a PR from `dev` into `master`
- [x] Write an ATM program
  - [x] Checkpoint: Create `assignment.rb`
  - [x] Checkpoint: Use [what we did in class](http://rpearce.gitbooks.io/ruby-june-2015/content/notes/1-2.html) as a reference
  - [x] prompt me for my name
    - [x] If I enter nothing, then it should continue to prompt me for my name (bonus for a witty message)
  - [x] prompt me for what I would like to do
    - [x] 1. Check my balance, 2. Withdraw Funds or 3. Cancel
    - [x] if I enter a number that is not within this list, it needs to continue to prompt me to enter either 1, 2 or 3 (this should not be hardcoded--use `.count` and a Range to programmatically figure out what numbers are acceptable)
    - [x] Check Balance
      - [x] print to the console `"Your balance is __$$$__"`
    - [x] Cancel
      - [x] print to the console `"Thank you and please come again."`
    - [x] Withdraw Funds
      - [x] prompt for how much money to withdraw
        - [x] if the amount is greater than the amount of money in the machine, print that the machine doesn't have that much money
        - [x] if the amount is greater than the amount of money the user has available, print that the user has insufficient funds
        - [x] if all is well, tell them to please take their money and then tell them their new balance
  - [x] Checkpoint: Used at least one method
  - [x] Checkpoint: Used at least one Range
  - [x] Checkpoint: Program can check more than 1 person's name & pin
  - [x] Complete: Pushed `assignment.rb

**Comment by rpearce** [comment]
I added an issue on your repo for a bug I found. Once that is resolved, this is done.

**Comment by rpearce** [comment]
I have a bit of code feedback, but without a PR to comment on, it will take me a while. If you haven't cleaned the issues up in the next PR, remind me to go through them with you next week.

**Comment by mcmillanator** [comment]
@rpearce I fixed the bug in 1.4, so I copy and paste that fix into 1.3.  I think I have a proper pull request for 1.4.  I also have one for 1.3 that I think is correct.  (link updated above)  If not just let me know, and I'll ask you about it this week.
